pre-commit.sh

```diff
@@ -6,3 +6,11 @@ if [ -z $AWX_IGNORE_BLACK ] ; then
         	exit 1)
 	fi
 fi
+
+if [ -z $AWX_IGNORE_USER ] ; then
+	if [ -d ./pre-commit-user ] ; then
+		for SCRIPT in `find ./pre-commit-user -name "*.sh" -executable` ; do
```

Comment:
A lot of the times I don't add a `.sh` extension to my scripts. It would also be cool if we could run any executable we put in this directory.

Reply:
It will now execute any executable file that is found.